node/src/components/storage.rs

```diff
@@ -1153,6 +1180,35 @@ fn insert_to_deploy_index(
     Ok(())
 }
 
+fn should_move_storage_files_to_network_subdir(root: &PathBuf, file_names: &[&str]) -> bool {
+    file_names
```

Comment:
This nice and neat, but unfortunately I think it'd be good to log a warning for the case where some but not all of these files exist.  The warning should give details of which files exist and which do not.